### DISCUSSION TITLE
能不能把添加app搞的容易一些

### DISCUSSION THREAD

**Discussion by dchw9956** [gkd-kit/gkd#341] [Ideas]
好多app没有适配无法创建规则,建议开发一个模块只用写下app名字

**Comment by lisonge** [comment]
没有适配怎么就不能创建规则了？

你直接在本地订阅界面输入APP规则创建不就行了吗？

**Comment by dchw9956** [comment]
那应该是我不会用吧，在试试看，已经适配过的APP可以在审查工具里生成规则适配给手机，列表里没有的APP即使把快照推送到电脑也没法生成规则



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年12月01日 22:40 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [gkd-kit/gkd] 能不能把添加app搞的容易一些 (Discussion #341) |

没有适配怎么就不能创建规则了？

**Reply by lisonge** [reply]
什么叫没法生成规则？是没有按钮吗？

**Comment by dchw9956** [comment]
不好意思，是我电脑的问题的问题。可以了，感谢你开发的软件



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年12月01日 23:08 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [gkd-kit] 能不能把添加app搞的容易一些 (Discussion #341) |

Closed #341 as duplicate.

**Comment by dchw9956** [comment]
碰到了一个问题不知道您有解决的办法没，快照用电脑打开同一个选择器有100多个，生成的规则会跳到别的地方



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年12月01日 23:08 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [gkd-kit] 能不能把添加app搞的容易一些 (Discussion #341) |

Closed #341 as duplicate.

**Reply by lisonge** [reply]
优化选择器的范围至 1 个

**Comment by dchw9956** [comment]
我已经按订阅的提示分享了，能力有限。只会简单使用，优化选择器不知道怎么用



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2023年12月04日 20:52 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [gkd-kit/gkd] 能不能把添加app搞的容易一些 (Discussion #341) |

优化选择器的范围至 1 个

**Reply by lisonge** [reply]
给出快照链接和说明需要点击的节点
